comp/netflow/server/server.go

```diff
@@ -30,29 +31,46 @@ type dependencies struct {
 	Hostname  hostname.Component
 }
 
+// Status represents the status of the server including details about
```

Comment:
You can run `inv go-lint` to run the linters manually; there's also a precommit hook you can enable that will lint everything before you commit, to avoid running afoul of these linters.